src/blueprints/Do.Blueprints.Service.Application/ExceptionHandling/Default/DefaultExceptionHandlingFeature.cs

```diff
@@ -10,11 +11,14 @@ public void Configure(LayerConfigurator configurator)
         configurator.ConfigureServiceCollection(services =>
         {
             services.AddSingleton<IExceptionHandler, HandledExceptionHandler>();
+
+            services.AddExceptionHandler<ExceptionHandlingMiddleware>();
+            services.AddProblemDetails();
         });
 
-        configurator.ConfigureMiddlewareCollection(middlewares =>
+        configurator.ConfigureMiddlewareCollection(middleware =>
         {
-            middlewares.Add<ExceptionHandlingMiddleware>(order: -20);
+            middleware.Add(app => app.UseExceptionHandler());
```

Comment:
there may be a better way to reach `app` than reaching it through `middlewares`

Reply:
I thought about adding a `LayerConfigurator` extension to configure `IApplicationBuilder` but wasn't sure about it

Reply:
i suppose there is already a layer that allows that configuration :eyes: let me check

Reply:
nope, there is not. looks like we've removed it. i'm resolving this one :raising_hand: